components/threading/virtualized_thread_viewer/thread_viewer_row.tsx

```diff
@@ -64,6 +67,15 @@ function ThreadViewerRow({
             />
         );
     default:
+        if (PostListUtils.isCombinedUserActivityPost(listId)) {
+            return (
+                <CombinedUserActivityPost
+                    combinedId={listId}
+                    previousPostId={getPreviousPostId}
+                    isLastPost={isLastPost}
+                />
+            );
+        }
```

Comment:
@koox00 I added this here to support `UserActivityPost` in the RHS.  Does this look ok to you? 

Should the "XXX added to the channel by you." be in the RHS, or in the center channel? It appears in the RHS now because the channel join request now includes the `rootId`, so the webapp considers it part of the thread.

An alternative fix would be to just send all `UserActivityPost` to the center channel instead.

Reply:
We should show it in the RHS, you might mention somebody from the global threads view